app/scripts/controllers/StudentsShowCtrl.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+define(['paw'], function(paw) {
+  paw.controller('StudentsShowCtrl', ['$routeParams', function($routeParams) {
+    var _this = this;
+
+    this.docket = $routeParams.docket; // For future Service calls
```

Comment:
Perfect! :+1: 

However, I wouldn't assign this to the controller, but to a variable, as it is not necessary to expose this information to the view.
i.e.: 

```js
var docket = $routeParams.docket; // For future Service calls
```